docs/source/requires.rst

```diff
@@ -56,82 +55,66 @@ Upstream Ubuntu Package Requirements
+distributions like CentOS 7.  However, disabling ``libsystemd`` means the
+user-space loaded PlatformIO interface won't access signals or controls from the
+GEOPM Systemd Service. The ``--disable-systemd`` option doesn't impact the GEOPM
+user unless the GEOPM Systemd Service is installed and active on the running
+Linux OS.
 
 Sphinx Requirement
 ^^^^^^^^^^^^^^^^^^
 
-The Sphinx python package is essential for generate man pages and HTML
-documentation. For the build to function correctly, the man pages must
-be generated. These man pages are included in the distribution tarball
-created by the ``make dist`` target, hence, building using this
-archive doesn't particularly require Sphinx. The requirement can be
-satisfied by PIP if there are issues installing the RPM packages for
-sphinx:
+The Sphinx python package is essential for generation of man pages and HTML
+documentation.  The source and build infrastrucure for this documentation is
+located in the ``docs`` subdirectory of the Git repository. The Sphinx
+requirement can be satisfied by PIP if there are issues installing the OS
+distribution packages:
 
 .. code-block:: bash
 
     python3 -m pip install --user sphinx sphinx_rtd_theme sphinxemoji
     export PATH=$HOME/.local/bin:$PATH
 
-These commands install Sphinx into your user's local Python packages
-and add the local Python package bin directory to your path for access
-to the sphinx-build script.
+These commands install Sphinx into your user's local Python packages and add the
+local Python package bin directory to your path for access to the
+``sphinx-build`` script.
 
-Run Requirements
-----------------
-
-There are runtime requirements for both the GEOPM Service and the
-GEOPM Runtime. These requirements relate to the MSR driver and the
-configuration of systemd.
-
-Access to MSRs enhances the capability of the GEOPM Service in terms
-of hardware telemetry and controls. While the GEOPM Service can
-function without access to MSRs, it provides a limited set of hardware
-features. For the GEOPM Runtime to function correctly, these
-MSR-related hardware features are necessary. Hence, MSR support is a
-hard requirement for the GEOPM Runtime.
-
-Erroneous settings for systemd can cause inter-process communication
-issues for both the GEOPM Service and the GEOPM Runtime.
 
 The MSR Driver
 ^^^^^^^^^^^^^^
 
-The msr-safe kernel driver provides two key features. Firstly, it
-offers a low latency interface for reading and writing many MSR values
-at once through an `ioctl(2)
-<https://man7.org/linux/man-pages/man2/ioctl.2.html>`_ system call,
-possibly improving the performance of GEOPM Runtime or other MSR
-usages.
+Access to MSRs enhances the capability of the GEOPM Access Service in terms of
+hardware telemetry and controls. While the GEOPM Service can function without
+access to MSRs, it provides a limited set of hardware features. For the GEOPM
+Runtime to function correctly, these MSR-related hardware features are
```

Comment:
Maybe emphasize **GEOPM Service** and **GEOPM Runtime** here to keep skim-readers from missing that this is a contrasting point.